packages/account/src/Components/trading-assessment/trading-assessment-dropdown.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Field } from 'formik';
+import { Dropdown } from '@deriv/components';
+
+const TradingAssessmentDropdownOption = ({ item_list, onChange, values, setFieldValue, setEnableNextSection }) => {
+    React.useEffect(() => {
+        checkIfAllFieldsFilled();
+    }, [values]);
+
+    const checkIfAllFieldsFilled = () => {
+        if (
+            values.cfd_trading_experience_mf &&
+            values.cfd_trading_frequency_mf &&
+            values.trading_experience_financial_instruments &&
+            values.trading_frequency_financial_instruments
+        ) {
+            setEnableNextSection(true);
```

Comment:
Can you directly use setEnableNextSection(your_condition_here) or create a variable with the condition and set the result to it?